### PR TITLE
Test covering non-latin chars & by default non latin word boundary

### PR DESCRIPTION
**Pull request by jcblw**
## Description

This adds in text for non latin character replacement, and also makes that the default path to avoid multiple implementations of replacement.

### CODE REVIEW

File: src/index.test.ts [hunk 1]
```diff
@@ -114,6 +114,18 @@ describe("Aracari", () => {
       .remap();
     expect(aracari.getText()).toBe("Done is the uno thing.");
   });
+  test("replaceText when passed an option of perserveWord option it should work with none latin characters", () => {
+    const element = document.createElement("div");
+    element.innerHTML = "<p>èstamos por la èsta hoy</p>";
```
Comment (jcblw) [inline src/index.test.ts]: @mike-goitia if you have a better test case let me know.

File: src/utils.ts [hunk 2]
```diff
@@ -14,3 +17,24 @@ export const escapeRegExp = (text: string) => {
...
+
+export const getSurroundingChars = (matchText: string): [string, string] => {
+  const leadingChar = wordBoundaryStartsWith.test(matchText)
+    ? matchText.charAt(0)
+    : "";
+  const endingChar = wordBoundaryEndsWith.test(matchText)
+    ? matchText.charAt(matchText.length - 1)
+    : "";
+  return [leadingChar, endingChar];
+};
```
Comment (mike-goitia) [inline src/utils.ts:40]: Beautiful ✨

File: src/index.test.ts (hunk 1 above)
Comment (mike-goitia) [reply src/index.test.ts]: might we try adding a test case for KO or Hindi Scripts? 우리는 오늘 여기에 있습니다 replace 여기에

**Review by mike-goitia — APPROVED** [review]

File: src/index.test.ts (hunk 1 above)
Comment (jcblw) [reply src/index.test.ts]: but for this we need to repeat word patterns like `èsta` in this test do you have an example of that?

Comment (mike-goitia) [reply src/index.test.ts]: Ah I see! 세계지도  <- 세계 World Map, World

**Comment by jcblw** [comment]
@mike-goitia updated the test to include the example you provided. Can I get a re-review?

**Review by mike-goitia — APPROVED** [review]
Maganda🔥

**Comment by jcblw** [comment]
published in [v0.5.0](https://www.npmjs.com/package/aracari/v/0.5.0)

**Pull request merged**